mathesar/templates/mathesar/login_base.html

```diff
@@ -192,5 +199,18 @@
   <div class="login-card align-center">
     <h1>{% block h1 %} {% endblock %}</h1>
     {% block box_content %} {% endblock %}
+    <form action="{% url 'set_language' %}" method="post">{% csrf_token %}
```

Comment:
Can you add comments where the `set_language` url coming from

Reply:
de03c76